README.md

```diff
@@ -43,9 +43,9 @@ T(n) = (3^i)T(n/(3^i)), i = log3n
      = n * T(n/(3^(log3n))
      = n(T(n/3^(log3n)))
      T(n/3^(log3n)) = 1
-     = 3^(log3n) * 1
-     = n 
-     = i = nlog3n ∈ Θ(n log n)
+     = n3^(log3n) * 1
```

Comment:
Somewhere you write, "since the addition is only + 1, and we ignore constants this has been taken off". You cannot ignore the constant at this stage of recurrence. You can only allow to drop the constant when you reach the final answer like O(3n) which is O(n). Here, you need to work on T(n) = 3T(n/3) + 1. Then, using substitution method to prove that.